extension/scripts/features/sorttravelmarket/ttSortTravelMarket.js

```diff
@@ -0,0 +1,184 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"Sort Market",
+		"travel",
+		() => settings.pages.travel.sortable,
+		null,
+		makeSortable,
+		removeSortable,
+		{
+			storage: ["settings.pages.travel.sortable"],
+		},
+		async () => {
+			await requireElement(".items-list-title");
```

Comment:
As told in the Discord before, those should go back in the `execute` function (`makeSortable`).